src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/EventValidationService.java

```diff
@@ -13,15 +14,18 @@
 @Service("eventValidationService")
 public class EventValidationService {
 
-    public List<String> validateReceiptDate(String receiptDate) {
+    public List<String> validateReceiptDate(CaseData caseData) {
 
         List<String> errors = new ArrayList<>();
 
-        LocalDate dateOfReceipt = LocalDate.parse(receiptDate);
+        LocalDate dateOfReceipt = LocalDate.parse(caseData.getReceiptDate());
 
         if (dateOfReceipt.isAfter(LocalDate.now())) {
             errors.add(FUTURE_RECEIPT_DATE_ERROR_MESSAGE);
         }
+        else{
+            caseData.setTargetHearingDate(dateOfReceipt.plusDays(TARGET_HEARING_DATE_INCREMENT).toString());
```

Comment:
I think if you don't return the caseData the setTargetHearingDate will not be populated

Reply:
I have tested this and double checked now it, i can cee the target hearing date being updated and displayed on the case details tab (as per image i uploaded into Jira). The casedata is passed byref is data updated should be availabe for the Response.